testCommon/src/testFixtures/java/com/virtuslab/gitmachete/testcommon/GitRepositoryBackedIntegrationTestSuiteInitializer.java

```diff
@@ -8,16 +8,16 @@
 
 import lombok.SneakyThrows;
 
-public abstract class BaseGitRepositoryBackedIntegrationTestSuite {
+public class GitRepositoryBackedIntegrationTestSuiteInitializer {
```

Comment:
Is it the final form of this class? or has sth been retained just for the sake of supporting junit4 in UI tests?

Reply:
Also... I'd simplify the name of this class. I'd argue that it theoretically doesn't need to be used to "back" any "integration test suite". It really just wraps a git repo prepared from the given script in a temp directory.

Reply:
before parameters were used to construct the test classes - I think that the parameters functionality was overused there. JUnit 5 more strictly requires that parameters are for test methods (cases). 

It is still needed for ui tests - that's true. but I think some improvements (esp naming) can be applied even though ui tests leave with junit 4 for now.

Reply:
Renamed to just `TestGitRepository`